### DISCUSSION TITLE
Add flugg/laravel-responder Support detailing response bodies and status codes

### DISCUSSION THREAD

**Discussion by agustin9998** [dedoc/scramble#215] [Ideas]
The package flugg/laravel-responder helps manage the jsonResponse in an application delivers, right now as laravel responder doesn't have any type hinting scramble reads it a "JsonResponse". 

It would be great to have extensions that handle this package.

What do you think?

**Comment by agustin9998** [comment]
I been working on these extensions and wanted to know if it's of interest for the package to include it as default

**Comment by romalytvynenko** [comment]
Hey @agustin9998

In general, Scramble's default will always be only Laravel's core stuff.

For everything else there will be extensions shipped as separate packages.

I'm working on making sure the API for such extensions allow to be easily installed and enabled for Scramble.

Thanks.
